### PR TITLE
edi_af_aisf_ash_kom, edi_af_falicility, edi_af_officer: Added some do…

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**
…cumentation

### CODE REVIEW

File: edi_af_aisf_ash_kom/__manifest__.py [hunk 1]
```diff
@@ -25,7 +25,10 @@
     'category': 'edi',
```
Comment (nils-vertel) [inline edi_af_aisf_ash_kom/__manifest__.py:25]: Update version and change log to
Change to new versioning scheeme

File: edi_af_officer/__manifest__.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
     'summary': 'EDI AF officers - support for officers',
     'licence': 'AGPL-3',
     'description': """ 
+Gets and sets up user and employee information from x500 AF Person API
```
Comment (nils-vertel) [inline edi_af_officer/__manifest__.py:29]: Update version to new version scheme

File: edi_af_officer/messages/af_get_officer.py [hunk 3]
```diff
@@ -55,7 +59,7 @@ def unpack(self):
                 if not office:
                     _logger.info("office number %s not in database, creating" % officer.get('officeCode'))
                     office = self.env['hr.department'].create({'name': officer.get('officeCode'), 'office_code': officer.get('officeCode'), 'note': _('Missing in AIS-F')})
-                if office and len(office) == 1:
+                if office and len(office) == 1: # hr.department should have a constraint added to it so length doesn't need to be checked
```
Comment (nils-vertel) [inline edi_af_officer/messages/af_get_officer.py:62]: Lots of commented out code, do we need to keep it?

**Review by nils-vertel — CHANGES_REQUESTED** [review]
Some minor things to fix

File: edi_af_officer/messages/af_get_officer.py (hunk 3 above)
Comment (SebastianGieser-Vertel) [reply edi_af_officer/messages/af_get_officer.py:62]: yes, it's unfinished code that hopefully may need to be finished in the future.

**Pull request merged**